### DISCUSSION TITLE
SiN waveguide design using AIM photonics 's PDK

### DISCUSSION THREAD

**Discussion by SadiMJawadAhsan** [gdsfactory/gdsfactory#2548] [Q&A]
This is going to be my first project using gdsfactory. I am looking for any tutorial to import AIM photonics PDK and designing a waveguide using gdsfactory. I found a tutorial on YouTube but that is using UBC PDK I think. Any help is welcome.

Thanks

**Comment by joamatab** [comment]
Hi,

I would not post any foundry information in public forums like this one, most foundries are very careful with their information to be accessible only under NDAs

therefore, as long as the PDKs are distributed under NDA we can't provide any tutorials in a public, you can contact the foundry to provide you with such tutorials
